metricflow/query/query_parser.py

```diff
@@ -43,6 +42,11 @@
     RequestTimeGranularityException,
 )
 
+logging.captureWarnings(True)
+import fuzzywuzzy.fuzz  # noqa: D
```

Comment:
I don't think this should be `noqa: D` as that's typically used for missing docstring errors (and it seems the D is interpreted as "all errors" which is very blah).  

I suspect this should be `# noqa: E402`